### DISCUSSION TITLE
Shall we change "Available Launchers" into "Invisible Launchers"?

### DISCUSSION THREAD

**Discussion by Nriver** [TriliumNext/Trilium#3518] [General]
Currently the launcher configuration groups the launchers by available ones and visible ones.

![launchers](https://user-images.githubusercontent.com/6752679/212216218-71bf6ef1-c4a3-4f43-90b9-9b44e20c6b08.png)

I think it's more intuitive to group the launchers into invisible ones and visible ones, and which would match the icons better.

![launchers2](https://user-images.githubusercontent.com/6752679/212217121-e49d241e-3035-485b-ac11-bdb50a533b93.png)

What do you think?

**Comment by zadam** [comment]
"Available" was motivated by thinking in use cases - it's meant to signal to the user that those are the launchers which user can choose from and drag into the visible subtree.

But I'm open to changes, let's perhaps wait for some more feedback.
